specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -3927,6 +3927,11 @@
           "$ref": "#/definitions/CurrentSpend",
           "readOnly": true
         },
+        "forecastSpend": {
+          "description": "The forecasted cost which is being tracked for a budget.",
+          "$ref": "#/definitions/ForecastSpend",
+          "readOnly": true
+        },
```

Comment:
Please add property at the end rather than in the middle

Reply:
Sure - added it at the end.